src/TxHistory/TxHistoryNavigator.tsx

```diff
@@ -1,13 +1,14 @@
 import {createStackNavigator} from '@react-navigation/stack'
-import React from 'react'
+import React, {useState} from 'react'
```

Comment:
was this a required change?

Reply:
Whats the preference. Could a lint rule be added?

Reply:
Done

Reply:
revert this change

Reply:
Reverted